skimage/feature/brief.py

```diff
@@ -182,3 +184,5 @@ def extract(self, image, keypoints):
 
         _brief_loop(image, self.descriptors.view(np.uint8), keypoints,
                     pos1, pos2)
+                    
+        return self
```

Comment:
missing Return in docs... also shall we return itself of just `self.descriptors`?